HOWTO-port-an-exercise.md

```diff
@@ -15,34 +15,38 @@ Here's a step-by-step procedure to port the **_foo-bar_** exercise:
     - add the labels: `x:action/create`, `x:module/practice-exercise`,
       `x:status/claimed`, `x:type/content`
     - assign it to yourself.
-1. copy the exercises/practice/foo-bar directory from the bash repo.
-1. in the exercises/practice/foo-bar directory, **delete** `foo_bar.sh`, and
+1. set `exercise=foo-bar`
+1. copy the exercises/practice/$exercise directory from the bash repo.
+1. in the exercises/practice/$exercise directory, **delete** `${exercise/-/_}.sh`, and
```

Comment:
Perhaps with backticks around the directories as well.

Reply:
Done